subsys/net/ip/6lo_private.h

```diff
@@ -81,6 +81,7 @@
 #define NET_6LO_NHC_UDP_PORT_10		0x02
 #define NET_6LO_NHC_UDP_PORT_11		0x03
 
+#define NET_6LO_NCH_UDP_CHECKSUM    0x04
```

Comment:
Is it typo?  "NCH", rest of the macros has NHC.

Reply:
Typo.